.github/workflows/go-getter.yml

```
@@ -0,0 +1,156 @@
+name: go-getter
+
+on: [push]
+
+env:
+  TEST_RESULTS_PATH: /tmp/test-results
+
+jobs:
+
+  linux-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version:
+          - 1.19
```

I suspect we'd be able to test `1.17`, `1.18`, and `1.19`, what do you think? 🤔 

```suggestion
        go-version:
          - 1.17
          - 1.18
          - 1.19
```